app/assets/stylesheets/custom.scss

```diff
@@ -291,3 +290,33 @@ ul, menu, dir {
 .filter {
   text-align: center;
 }
+
+
+// /* =========================
+//
+//      USER LISTINGS INDEX PAGE
+//
+//    =========================*/
+
+#img_container {
+    position:relative;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered display, position, text-align<br>Colon after property should be followed by one space